scss/typography/_helpers.scss

```diff
@@ -75,4 +98,16 @@ $stat-font-size: 2.5rem !default;
     margin-#{$global-left}: 0;
     list-style: none;
   }
+
+  .dropcaps {
+    line-height: $dropcaps-line-height;
+    &:first-letter {
+      margin-top: $dropcaps-first-letter-margin-top;
+      margin-bottom: $dropcaps-first-letter-margin-bottom;
+      padding-right: $dropcaps-first-letter-padding-right;
+      font-size: $dropcaps-first-letter-font-size;
+      line-height: $dropcaps-first-letter-line-height;
+      float: $dropcaps-first-letter-float;
+    }
+  }
```

Comment:
Please create a **Sass component** with a mixin:
```scss
@mixin dropcaps() { // we have no options for now, unless you want to provide `$dropcaps-lines` or `$dropcaps-space` now.
   ...code...
}
```

Then use it to create the **CSS component**:
```scss
.dropcaps {
    @include dropcaps;
}
```